component/src/test/java/io/siddhi/extension/io/nats/utils/NATSClient.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
The license year is added only when the file is added, not for every change. Revert

Reply:
Fixed in https://github.com/siddhi-io/siddhi-io-nats/pull/37/commits/867fce4dac46cc0371b9dd2dd32807f83c541684